src/Umbraco.Web.BackOffice/Filters/ValidateAngularAntiForgeryTokenAttribute.cs

```diff
@@ -44,15 +44,6 @@ public ValidateAngularAntiForgeryTokenFilter(ILogger<ValidateAngularAntiForgeryT
 
             public async Task OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
             {
-                if (context.Controller is ControllerBase controller && controller.User.Identity is ClaimsIdentity userIdentity)
```

Comment:
This is a check to ensure that any endpoint that requires anti forgery done is actually performed when the authentication ticket is based on a cookie. Without this check it means that other auth types like Bearer token auth that does not rely on cookies will fail the anti forgery check.

Reply:
Ah i see, then we of course can't remove it completely. I've reintroduced it, adding a return statement after the `next()` call, if I understand it correctly, we want to entirely skip this middleware if there is no cookie since it's dependant upon it?